Source/Core/Common/StringUtil.cpp

```diff
@@ -664,3 +665,19 @@ std::vector<std::string> CommandLineToUtf8Argv(const wchar_t* command_line)
   return argv;
 }
 #endif
+
+std::string GetEscapedHtml(std::string html)
+{
+  static constexpr std::array<std::array<const char*, 2>, 5> replacements{
```

Comment:
```cpp
static constexpr std::array<std::array<const char*, 2>, 5> replacements{{
    {{"&", "&amp;"}},
    {{"<", "&lt;"}},
    {{">", "&gt;"}},
    {{"\"", "&quot;"}},
    {{"'", "&apos;"}},
}};
```

Does that work? (It seems to compile fine in godbolt for GCC >=9 and Clang >= 9 at least)

Reply:
Yes it does.